.scss-lint.yml

```diff
@@ -0,0 +1,17 @@
+linters:
+  Comment:
+    enabled: false
+  HexLength:
+    style: "long"
+  HexNotation:
+    style: "uppercase"
+  IdSelector:
+    enabled: false
+  ImportantRule:  # While I agree with this one, we do need to use !important in some cases
```

Comment:
Is there maybe a way to exclude certain lines from the linting, or would that be overkill?

Reply:
You can exclude blocks, but not specific lines